parsers/s01-parse/PintjesB/technitium-logs.yaml

```diff
@@ -0,0 +1,25 @@
+name: PintjesB/technitium-logs
+description: "Parse Technitium auth logs"
+onsuccess: next_stage
+filter: evt.Parsed.program == 'technitium'
+pattern_syntax:
+  DATETIME_CUSTOM: '%{YEAR:year}-%{MONTHNUM:month}-%{MONTHDAY:day} %{TIME:time} %{TZ}'
```

Comment:
Could we call this `technitium_DATETIME` so it may not clash with other parsers as custom are shared.